pkg/controller/route/routecontroller.go

```diff
@@ -23,9 +23,19 @@ import (
 
 	"github.com/golang/glog"
 	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/cache"
 	client "k8s.io/kubernetes/pkg/client/unversioned"
 	"k8s.io/kubernetes/pkg/cloudprovider"
+	"k8s.io/kubernetes/pkg/controller"
+	"k8s.io/kubernetes/pkg/controller/framework"
+	"k8s.io/kubernetes/pkg/runtime"
 	"k8s.io/kubernetes/pkg/util"
+	"k8s.io/kubernetes/pkg/util/workqueue"
+	"k8s.io/kubernetes/pkg/watch"
+)
+
+const (
+	workerGoroutines = 5
```

Comment:
This should be a flag, just like ReplicationController/ EndpointController/ DeploymentController do  https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/options/options.go#L141-L144
I will send a PR to make this configurable once this PR get merged . Current, add a flag to ControllerManager/Scheduler/kubelet need update auto generated files, so I think it's better to send an another PR to address the flag.